lib/membrane_element_file/sink_multi/split.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Membrane.Element.File.Sink.Multi.Split do
```

Comment:
That name is horribly long. How about renaming it to `Membrane.Element.File.SplitEvent` and moving to proper catalogue?